WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -3408,60 +3434,72 @@ else
-          { spell = 265931, type = "debuff", unit = "target"}, -- Conflagrate
+          { spell = 172, type = "debuff", unit = "target"}, -- Coruption
+          { spell = 348, type = "debuff", unit = "target"}, -- Immolate
+          { spell = 702, type = "debuff", unit = "target"}, -- Curse of Weakness
+          { spell = 710, type = "debuff", unit = "multi"}, -- Banish
+          { spell = 1714, type = "debuff", unit = "target"}, -- Curse of Tongues
+          { spell = 1098, type = "debuff", unit = "multi"}, -- Enslave Demon
+          { spell = 5782, type = "debuff", unit = "target"}, -- Fear
+          { spell = 6358, type = "debuff", unit = "target"}, -- Seduction
+          { spell = 6360, type = "debuff", unit = "target"}, -- Whiplash
+          { spell = 6789, type = "debuff", unit = "target", talent = 14 }, -- Mortal Coil
           { spell = 17735, type = "debuff", unit = "target"}, -- Suffering
-          { spell = 118699, type = "debuff", unit = "target"}, -- Fear
+          { spell = 22703, type = "debuff", unit = "target"}, -- Infernal Awakening
+          { spell = 30283, type = "debuff", unit = "target"}, -- Shadowfury
           { spell = 80240, type = "debuff", unit = "target"}, -- Havoc
-          { spell = 6789, type = "debuff", unit = "target", talent = 14 }, -- Mortal Coil
+          { spell = 118699, type = "debuff", unit = "target"}, -- Fear
+          { spell = 157736, type = "debuff", unit = "target"}, -- Immolate
           { spell = 196414, type = "debuff", unit = "target", talent = 2 }, -- Eradication
           { spell = 234153, type = "debuff", unit = "target"}, -- Drain Life
-          { spell = 6360, type = "debuff", unit = "target"}, -- Whiplash
-          { spell = 30283, type = "debuff", unit = "target"}, -- Shadowfury
-          { spell = 710, type = "debuff", unit = "multi"}, -- Banish
-          { spell = 1098, type = "debuff", unit = "multi"}, -- Enslave Demon
-          { spell = 6358, type = "debuff", unit = "target"}, -- Seduction
-
+          { spell = 265931, type = "debuff", unit = "target"}, -- Conflagrate
+          { spell = 334275, type = "debuff", unit = "target"}, -- Curse of Exhaustion
         },
         icon = 135817
       },
       [3] = {
         title = L["Abilities"],
         args = {
+          { spell = 126, type = "ability"}, -- Eye of Kilrogg
+          { spell = 172, type = "ability", requiresTarget = true, debuff = true}, -- Corruption
           { spell = 348, type = "ability", requiresTarget = true, debuff = true}, -- Immolate
+          { spell = 686, type = "ability"}, -- Incinerate
           { spell = 698, type = "ability"}, -- Ritual of Summoning
+          { spell = 702, type = "ability", requiresTarget = true, debuff = true}, -- Curse of Weakness
           { spell = 710, type = "ability", requiresTarget = true, debuff = true}, -- Banish
+          { spell = 1089, type = "ability"}, -- Subhugate Demon
```

Comment:
![image](https://user-images.githubusercontent.com/189656/95002155-4b9fce00-05d1-11eb-9790-05eaa0323edb.png)
i can't find a talent for this in destruction spec